apps/daimo-mobile/src/view/shared/SwipeUpDown.tsx

```diff
@@ -81,22 +80,32 @@ export const SwipeUpDown = forwardRef<SwipeUpDownRef, SwipeUpDownProps>(
     });
 
     const [isMini, setIsMini] = useState(true);
+    const historyOpOpacity = useSharedValue(0);
 
     // When user selects a transaction, open the bottom sheet part way.
     const animatedIndex = useSharedValue(0);
     const sheetExpand = () => {
       if (animatedIndex.value === 1) {
         bottomRef.current?.snapToIndex(2);
       }
-      setSnapPoints([posYMini, 450, posYFull]);
-      if (animatedIndex.value === 0) {
-        bottomRef.current?.snapToIndex(1);
-      }
+      snapPoints.value = [posYMini, 450, posYFull];
+      // skip one frame to let snapPoints addapt
```

Comment:
what do you mean by by "let snapPoints adapt"

Reply:
When I change the `snapPoints.value` it changes but with slight delay, so when I tried to call `snapToIndex` just after this line without timeout it still snapped to old snapPoints 😔

I will try to make it somehow else so it doesnt use hardcoded timeout

Reply:
Used useAnimatedReaction instead and seems to work fine 🙏